assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -452,6 +450,9 @@
 						$.removeData( this, pluginName );
 					}
 
+					return;
+				} else if ( 'destroy' === options ) {
```

Comment:
Why do you need this?

Reply:
I wanted to make sure that we can delete the droppable for both the container and the container inner, without storing that data in a setting. 

Destroying an unitialised container was causing an error. I have escaped that now.